crates/trycmd/src/schema.rs

```diff
@@ -559,7 +559,7 @@ impl From<OneShot> for TryCmd {
                 expected_stderr_source: None,
                 expected_stderr: stderr.map(crate::Data::text),
                 binary,
-                timeout,
+                timeout: timeout.map(|e| e.unsigned_abs()),
```

Comment:
Why are we taking an absolute value?

Reply:
From what I understand, there is no equivalent to `SignedDuration` in the standard lib, because `std::time::Duration` does not have a sign

I don't want to paraphase, so I would refer to the library explanation

https://github.com/BurntSushi/jiff/blob/4404fb0f6b5b48b8177546eaaef91acf603a2736/DESIGN.md?plain=1#L268

Reply:
This might be another case where you want `std::time::Duration::try_from(timeout).unwrap()`. Or even `std::time::Duration::try_from(timeout).ok()` could be justifiable.

Reply:
Nice, updated